### PR TITLE
Added inline-flex to the search form and redirected the homepage to dashboard/login.

### PR DESCRIPTION
**Pull request by mihaelavalac**
Completed #4

### CODE REVIEW

File: resources/views/layouts/navigation.blade.php [hunk 1]
```diff
@@ -20,25 +20,22 @@
...
 
-                                
-                                    <button type="submit" class="btn inline-block px-6 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
+
+                                    <button type="submit" class="btn px-4 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
```
Comment (ryvrook) [inline resources/views/layouts/navigation.blade.php:31]: Good Style change. Inline-block removed.

File: routes/web.php [hunk 2]
```diff
@@ -15,25 +15,19 @@
...
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
+//     return view('welcome');
+// });
 
-Route::get('/dashboard', function () {
+Route::get('/', function () {
```
Comment (ryvrook) [inline routes/web.php:22]: Made base view dashboard view instead of having the link.  (/dashboard)

File: routes/web.php [hunk 3]
```diff
@@ -15,25 +15,19 @@
...
-/*Route::controller(GameController::class)->group(function(){
-    Route::get('/games', 'show');
-});
-*/
 
-// Route::get('/games', [GameController::class, 'show'])
-// ->name('games');
+Route::get('/dashboard', function () {
+    return view('dashboard');
+})->middleware(['auth'])->name('dashboard');
```
Comment (ryvrook) [inline routes/web.php:29]: Route addition.

**Review by ryvrook — APPROVED** [review]
Good changes, and curious why this is done on the wishlist branch?

File: resources/views/layouts/navigation.blade.php (hunk 1 above)
Comment (mihaelavalac) [reply resources/views/layouts/navigation.blade.php:31]: I was planning on working on the wishlist and after creating the branch I saw that Brian is working on it. I made these other changes and pushed them without realizing that the branch doesn't match.

File: routes/web.php (hunk 2 above)
Comment (mihaelavalac) [reply routes/web.php:22]: Make sense.

**Pull request merged**